package/entrypoint.sh

```diff
@@ -6,6 +6,14 @@ if [ ! -e /run/secrets/kubernetes.io/serviceaccount ] && [ ! -e /dev/kmsg ]; the
     exit 1
 fi
 rm -f /var/lib/rancher/k3s/server/cred/node-passwd
+if [ -e /var/lib/rancher/etcd ] && [ ! -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  mkdir -p /var/lib/rancher/k3s/server/db
+  ln -sf /var/lib/rancher/etcd /var/lib/rancher/k3s/server/db/etcd
+  echo -n 'default' > /var/lib/rancher/k3s/server/db/etcd/name
+fi
+if [ -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  k3s server --cluster-init --cluster-reset &> ./k3s-cluster-reset.log
```

Comment:
What is the reason to suppress the log here? If this fails, the container log is empty and the user does not know what's going on.

And do we need any counter to when this fails to recover automatically or if this fails, it needs manual intervention and we need to point that out? Or is that covered when we show the log from `--cluster-reset`?

Reply:
The log from this operation is quite noisy, and will not normally be something that the user needs to see. We could probably check to see if it succeeds, and cat the output if it fails?

Reply:
@superseb Is this ok with you?

Reply:
Yeah we dont need more noisy stuff in there but we can't not let the user know whats going on in this case. Exit code check and showing the log is fine.

Does a `cluster-reset` failing for some reason always require manual intervention? So we don't need some sort of auto recovery in that case and just always manual intervention where the user can use the printed log? I'm asking this in relation to how it was, and that was k3s always trying to start (it didn't do a cluster reset afaik)

Reply:
The cluster-reset isn't really for k3s, it's for etcd. Since we can't tell k3s to use the loopback address for the cluster member IP, we need to reset the cluster member address to the current container address every time, or etcd will refuse to start.

https://github.com/rancher/rancher/pull/35323#discussion_r747068862